src/registrar/forms/user_profile.py

```diff
@@ -93,4 +93,10 @@ def __init__(self, *args, **kwargs):
         self.fields["title"].label = "Title or role in your organization"
 
         # Define the "full_name" value
-        self.fields["full_name"].initial = self.instance.get_formatted_name()
+        full_name = ""
+        if self.instance.first_name and self.instance.last_name:
```

Comment:
Seems like this logic should be inside of get_formatted_name(). What was the reasoning for doing these checks here?

Reply:
Got it...seems like an inconsistent UI rule, but I won't argue the ACs this time around.